src/binary-reader-objdump.cc

```diff
@@ -296,6 +297,16 @@ Result BinaryReaderObjdumpDisassemble::OnOpcodeBare() {
   return Result::Ok;
 }
 
+Result BinaryReaderObjdumpDisassemble::OnOpcodeIndex(Index value) {
```

Comment:
I was trying to avoid having too many callbacks here, but maybe you are right.. let me see.